package.json

```diff
@@ -16,7 +16,7 @@
   },
   "dependencies": {
     "dotenv": "^14.3.2",
-    "express": "^4.17.2",
+    "express": "^4.18.2",
```

Comment:
why did you change the version of express?

Reply:
I didn't change the version of express.

Reply:
Yes you did, you might don't know how did you do it but you did. because it shows into your PR that you changed the version from 4.17.2 to 4.18.2